main/php_glob.c

```diff
@@ -37,24 +37,29 @@
  *
  * Optional extra services, controlled by flags not defined by POSIX:
  *
- * GLOB_QUOTE:
+ * PHP_GLOB_QUOTE:
  *	Escaping convention: \ inhibits any special meaning the following
  *	character might have (except \ at end of string is retained).
- * GLOB_MAGCHAR:
+ * PHP_GLOB_MAGCHAR:
  *	Set in gl_flags if pattern contained a globbing character.
- * GLOB_NOMAGIC:
- *	Same as GLOB_NOCHECK, but it will only append pattern if it did
+ * PHP_GLOB_NOMAGIC:
+ *	Same as PHP_GLOB_NOCHECK, but it will only append pattern if it did
  *	not contain any magic characters.  [Used in csh style globbing]
- * GLOB_ALTDIRFUNC:
+ * PHP_GLOB_ALTDIRFUNC:
  *	Use alternately specified directory access functions.
- * GLOB_TILDE:
+ * PHP_GLOB_TILDE:
  *	expand ~user/foo to the /home/dir/of/user/foo
- * GLOB_BRACE:
+ * PHP_GLOB_BRACE:
  *	expand {1,2}{a,b} to 1a 1b 2a 2b
  * gl_matchc:
  *	Number of matches in the current invocation of glob.
  */
 
+#include "php_glob.h"
+
+#if defined(HAVE_GLOB) && defined(PHP_SYSTEM_GLOB)
+#else
```

Comment:
I'm a bit confused by this if/else

Reply:
This should basically be an empty file if we're using system glob; I should have invented the if here, or if we have one, missing a thing that prevents compiler warnings on empty files or something.

Reply:
Okay, new code makes sense :)